sources/assets/zsh/aliases.d/ysoserial

```diff
@@ -1 +1 @@
-alias ysoserial='JAVA_HOME=/usr/lib/jvm/java-11-openjdk java -jar /opt/tools/ysoserial/ysoserial.jar'
+alias ysoserial='JAVA_HOME=/usr/lib/jvm/java-11-openjdk java -jar /opt/tools/ysoserial/ysoserial.jar --add-opens=java.xml/com.sun.org.apache.xalan.internal.xsltc.trax=ALL-UNNAMED --add-opens=java.xml/com.sun.org.apache.xalan.internal.xsltc.runtime=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED'
```

Comment:
not sure this is okay, from what I read on https://github.com/ThePorgs/Exegol-images/issues/234 and https://github.com/frohoff/ysoserial/issues/203#issuecomment-1691067536, the option must be placed like `java <options> -jar ./ysoserial.jar <payload> <command>`